rpcs3/Emu/Io/usio.cpp

```diff
@@ -389,20 +403,14 @@ void usb_device_usio::usio_read(u8 channel, u16 reg, u16 size)
 			{
 			case 0x0000:
 			{
-				ensure(size == 0xB8);
-				std::vector<u8> buffer[3];
-				for (int i = 0; i < 3; i++)
-				{
-					buffer[i].resize(i < 2 ? 64 : 56);
-					memcpy(buffer[i].data(), g_fxo->get<usio_memory>().backup_memory.data() + i * 64, buffer[i].size());
-					q_replies.push(buffer[i]);
-				}
+				ensure(size <= 0xB8);
```

Comment:
Do we really need an assertation for this and the other case? if it's already handling variable size may log an error instead and continue execution. This way the developer is notified with unknown behavior but it still works.

Reply:
> Do we really need an assertation for this and the other case? if it's already handling variable size may log an error instead and continue execution. This way the developer is notified with unknown behavior but it still works.

I'm not pretty sure. The person who initially made the usio.cpp put ensure(size) in almost every read/write case.
If you think its unnecessary, maybe we can remove it in the future?

Reply:
Maybe, log an error for now.

Reply:
> Maybe, log an error for now.

I think this PR can be merged as-is, and I'll do that in the near future in another PR.